test/utils-test.ts

```diff
@@ -30,4 +30,10 @@ describe('WSL path format to Windows', () => {
     const windows = 'D:\\Users\\user1\\AppData';
     assert.equal(toWinDirFormat(wsl), windows);
   });
+
+  it('getLocalAppDataPath returns a correct path', () => {
+    process.env.PATH = '/mnt/c/Users/user1/.bin:/mnt/c/Users/user1/AppData/';
+    const appDataPath = '/mnt/c/Users/user1/AppData/Local';
```

Comment:
ah, it was spanning the separator before? how about using `[^:]` then? 

I'm not sure if the original intention was to allow spanning multiple directories deep AppData

Reply:
we don't need to span multiple directories deep so should I change this?

Reply:
discussed above, seems good to account for both :)